lib/main.coffee

```diff
@@ -93,35 +90,15 @@ module.exports =
 
   # Public: Creates AutocompleteManager instances for all active and future editors (soon, just a single AutocompleteManager)
   activate: ->
-    @getAutocompleteManager()
-    # @activateTimeout = setTimeout(@getAutocompleteManager, 0)
+    run = =>
+      @getAutocompleteManager()
+    setTimeout(run.bind(this), 0)
```

Comment:
I think we should try to make this fast in the sync case. Even if there is a method that does all the setup. The method could be called by the tests, or by some event the user triggers.